iucnredlist.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: 90321148-fedc-4494-81df-fce916b15f2d
```

Comment:
Ah, I wondered what `ProjectId` was but looks like it's a new addition in a recent RStudio release and won't change once set - unless a dev is on an older version of RStudio (note to self - update RStudio).
https://forum.posit.co/t/use-for-projectid-in-rproj-file/196345
https://github.com/rstudio/rstudio/pull/15140
(fyi @simon-tarr)